### ISSUE TITLE
Package is not exporting a proper types for ESM, unable to use with typescript set to NodeNext module and module resolution.

### ISSUE THREAD

**Issue by michalkvasnicak** [jshemas/openGraphScraper#224]
**Describe the bug**

Package does not have correct typings. When you set typescript to `"module": "NodeNext", "moduleResolution": "NodeNext"` it fails to compile.

**To Reproduce**

https://stackblitz.com/edit/vitejs-vite-dj4sv9?file=src%2Findex.ts

**Expected behavior**

Should import the package without any issues.

**Actual behavior**

Fails with an error:

```
Type 'typeof import("stackblitz:/node_modules/open-graph-scraper/dist/index")' has no call signatures.(2349)
```

**Additional context**

 - Node Version: 18.20.3
 - openGraphScraper Version: 6.5.2

**Comment by jshemas** [comment]
Okay, i'm able to see the issue now locally. 

I still see the error even if I build a `dist` for ESM, so I will need to dig into some more. In the meantime you can use `ogs.default();` to fix the issue.

**Comment by jshemas** [comment]
Hello,

This should be fixed in `open-graph-scraper@6.6.0`. I was able to get your code example working with my new build.

![image](https://github.com/jshemas/openGraphScraper/assets/2013365/02ebc74b-848c-404a-ab9b-2558ef3b3bd3)

**Comment by jshemas** [comment]
Closing this for now. Please open a new issue if you have any other problems.
